src/bin/pg_autoctl/cli_create_drop_node.c

```diff
@@ -33,19 +33,24 @@
 #include "primary_standby.h"
 
 MonitorConfig monitorOptions;
+bool dropAndDestroy = false;
```

Comment:
not a fun of global stuff. can we at least make it static ?

Reply:
Done, thanks!